### ISSUE TITLE
`@protected` and `@private` will be ignored

### ISSUE THREAD

**Issue by raabf** [Feneric/doxypypy#21]
Actually you add the `@protected` and `@private` commands if the function has an single underscore `_` respectively double underscore `__`.
The interpretation of the underscores is correct but [doxygen will ignore them](http://www.stack.nl/~dimitri/doxygen/manual/commands.html#cmdprotected) if the language is object-oriented:

> \protected
> Indicates that the member documented by the comment block is protected, i.e., should only be accessed by other members in the same or derived classes.
>
> Note that Doxygen automatically detects the protection level of members in object-oriented languages. This command is intended for use only when the language does not support the concept of protection level natively (e.g. C, PHP 4).

(same for private)

doxygen actually mark everything as private which begins with one or more underscore.
Of course you can nothing do to change that, I want to only leave this information here for users which are confused (and that you do not have to add the `@protected` and `@private` commands).
I opened an [issue](https://bugzilla.gnome.org/show_bug.cgi?id=748753) so that it will be corrected.

**Comment by Feneric** [comment]
Yeah, over the years a few other similar issues have been raised but there's not too much we can do about them. Our policy has been to make doxypypy send the best data that it can even knowing it may be ignored by the current version of Doxygen. If at some point that data gets considered we'll be ready.
